desktop/webpack.config.base.js

```diff
@@ -24,11 +24,7 @@ module.exports = {
       loader: 'json',
     }, {
       test: /\.(gif|png)$/,
-      loader: 'file',
```

Comment:
this is pretty slow so i removed it. less safe but takes half the time to webpack build